usr/share/rear/output/default/94_grub2_rescue.sh

```diff
@@ -11,7 +11,10 @@ is_true "$GRUB_RESCUE" || return
 # (grub-probe or grub2-probe only exist in GRUB 2)
 # in particular do not run this script when GRUB Legacy is used
 # (for GRUB Legacy output/default/94_grub_rescue.sh is run):
-type -p grub-probe >&2 || type -p grub2-probe >&2 || { LogPrint "Skipping GRUB_RESCUE setup for GRUB 2 (no GRUB 2 found)." ; return ; }
+if [[ ! $( type -p grub-probe ) && ! $( type -p grub2-probe ) ]]; then
```

Comment:
Oh - yes - I should have mentioned
that I always run it as "rear -d -D ..."

I simply ignore the log (when things worked well)
but when something failed or looked suspicious
then I need the log with debug info already in it.

Cf. "Debugging issues with Relax-and-Recover (rear)"
in https://en.opensuse.org/SDB:Disaster_Recovery

But currently I miss stdout in the log because I woul need
that to get the complete picture in the log. Therefore
I did https://github.com/rear/rear/issues/885